tensorflow/contrib/BUILD

```diff
@@ -103,8 +105,9 @@ py_library(
     ] + if_mpi(["//tensorflow/contrib/mpi_collectives:mpi_ops_py"]),
 )
 
-cc_library(
+tf_cc_library(
     name = "contrib_kernels",
+    is_external = True,
```

Comment:
Is there a way to avoid having `is_external = True` around a bunch of rules? Again, my target is maintainability here. It's fine if it's only needed for a few rules, but it's really hard for people to reason about when they should and shouldn't add it.

Reply:
I didn't find how to do this. 
This argument depends on how will you use this library.
The rule is: 
- If this library is a part of \_pywrap_tensorflow_internal.so, please set it to **False**.
- If this library will be used in a cc_binary, which is dynamic linked to \_pywrap_tensorflow_internal.so, please set it to **True**.
- If this library is only available on non-Windows systems(e.g.Linux), either value is ok.

Itself doesn't know such information.